src/components/ModelTableList/shared.tsx

```diff
@@ -75,7 +76,11 @@ export const generateTableHeaders = (
 ) => {
   const rows = [
     {
-      content: <Status status={label} count={count} />,
+      content: options?.showHeaderStatus ? (
+        <Status status={label} count={count} />
+      ) : (
+        `${label} (${count})`
```

Comment:
This was displaying the status icon for all tables instead of just the status tables.